docs/sources/tempo/traceql/_index.md

```diff
@@ -119,6 +119,26 @@ For example, to find traces with an attribute of `sla` set to `critical`:
 { .sla = "critical" }
 ```
 
+### Quoted attribute names
+
+Attribute names can contain terminal characters, such as a period (`.`). To search span attributes with terminal characters, you can use quoted attribute syntax. A quoted attribute should be enclosed inside double quotes, for example, `"example one"`. All characters between the quotes are considered part of the attribute name.
```

Comment:
nit: [Semantic line breaks](https://sembr.org/) are better for future line based diffing in the GitHub UI and result in the same HTML output.

```suggestion
Attribute names can contain terminal characters, such as a period (`.`).
To search span attributes with terminal characters, you can use quoted attribute syntax.
A quoted attribute should be enclosed inside double quotes, for example, `"example one"`.
All characters between the quotes are considered part of the attribute name.
```